### ISSUE TITLE
[CIS Azure] App Service rule 9.1 needs extra permissions

### ISSUE THREAD

**Issue by kubasobon** [elastic/cloudbeat#1974]
**Describe the bug**
Follow-up to https://github.com/elastic/cloudbeat/issues/1758

Cloudbeat lacks permissions to access `Microsoft.Web/sites/*/config`.

```
[elastic_agent.cloudbeat][error] Error running fetcher for key azure_cloud_assets_fetcher: error while enriching assets: error while retrieving AppService AuthSettings: POST https://management.azure.com/subscriptions/ef111ee2-6c89-4b09-92c6-5c2321f888df/resourceGroups/azurecloudbeatcitests/providers/Microsoft.Web/sites/test-app-service-fail/config/authsettings/list
--------------------------------------------------------------------------------
RESPONSE 403: 403 Forbidden
ERROR CODE: AuthorizationFailed
--------------------------------------------------------------------------------
{
  "error": {
    "code": "AuthorizationFailed",
    "message": "The client '2d198592-b967-4752-b286-b2624d5cc8e6' with object id '2d198592-b967-4752-b286-b2624d5cc8e6' does not have authorization to perform action 'Microsoft.Web/sites/config/list/action' over scope '/subscriptions/ef111ee2-6c89-4b09-92c6-5c2321f888df/resourceGroups/azurecloudbeatcitests/providers/Microsoft.Web/sites/test-app-service-fail/config/authsettings' or the scope is invalid. If access was recently granted, please refresh your credentials."
  }
}
--------------------------------------------------------------------------------

error while retrieving AppService AuthSettings: POST https://management.azure.com/subscriptions/ef111ee2-6c89-4b09-92c6-5c2321f888df/resourceGroups/azurecloudbeatcitests/providers/Microsoft.Web/sites/test-app-service-pass/config/authsettings/list
--------------------------------------------------------------------------------
RESPONSE 403: 403 Forbidden
ERROR CODE: AuthorizationFailed
--------------------------------------------------------------------------------
{
  "error": {
    "code": "AuthorizationFailed",
    "message": "The client '2d198592-b967-4752-b286-b2624d5cc8e6' with object id '2d198592-b967-4752-b286-b2624d5cc8e6' does not have authorization to perform action 'Microsoft.Web/sites/config/list/action' over scope '/subscriptions/ef111ee2-6c89-4b09-92c6-5c2321f888df/resourceGroups/azurecloudbeatcitests/providers/Microsoft.Web/sites/test-app-service-pass/config/authsettings' or the scope is invalid. If access was recently granted, please refresh your credentials."
  }
}
--------------------------------------------------------------------------------
```

### ToDo

Add necessary permissions to ARM template.

**Comment by kfirpeled** [comment]
Synced with @oren-zohar and we agreed that this should be backported to 8.13 and will be tested next BC

**Comment by kubasobon** [comment]
Merged to main, backported and released as part of 8.13.

**Comment by Omolola-Akinleye** [comment]
Verified
<img width="1242" alt="image" src="https://github.com/elastic/cloudbeat/assets/17135495/ad128a5d-6560-431b-95af-ab40519fd3ad">
